test/uk/gov/hmrc/cgtpropertydisposals/controllers/actions/AuthenticatedActionsSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.controllers.actions
+
+import akka.util.Timeout
+import org.mockito.{MockitoSugar, _}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FlatSpec, Matchers}
+import play.api.mvc.{BodyParsers, Results}
+import play.api.test.{FakeRequest, Helpers, NoMaterializer}
+import uk.gov.hmrc.auth.core.retrieve.{Credentials, ~}
+import uk.gov.hmrc.auth.core.{AuthConnector, BearerTokenExpired, MissingBearerToken, SessionRecordNotFound}
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+
+class AuthenticatedActionsSpec
+    extends FlatSpec
+    with Matchers
+    with MockitoSugar
```

Comment:
it's a bit weird having mockito being used in this one place when scalamock is being used everywhere else

Reply:
Do you want me to convert my tests to Scalamock?

Reply:
it'd make the test code more consistent I think if you did - then we could get rid of the mockito library in our `AppDependencies` as well